hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CopyCommands.java

```diff
@@ -256,6 +258,8 @@ protected void processOptions(LinkedList<String> args)
         "  -p : Preserves timestamps, ownership and the mode.\n" +
         "  -f : Overwrites the destination if it already exists.\n" +
         "  -t <thread count> : Number of threads to be used, default is 1.\n" +
+        "  -q <threadPool size> : Number of threadPool queue size to be used, " +
```

Comment:
Just a nit: remove "Number of"

Reply:
Thanks to @jojochuang  comment, I will make some changes later.